webpack.common.js

```diff
@@ -38,7 +38,7 @@ module.exports = {
     htmlPlugin,
     new CleanWebPackPlugin(['build']),
     new webpack.HotModuleReplacementPlugin(),
-    new BundleAnalyzerPlugin(),
+    // new BundleAnalyzerPlugin(),
```

Comment:
I suggest to make a switch be `config` or `environment variable` instead of comment here. e.g.

```suggestion
plugins = [ ... ]

if (process.env.ANALYNE === 'true') {
  plugins.push(new BundleAnalyzerPlugin())
}
```

```js
"scripts": {
  "build:analyze": "ANALYNE=true webpack --config webpack.prod.js",
}
```

Reply:
Ok, I don't know I can pass value by shell command. Thanks to the advice.